server/routes/user.js

```diff
@@ -1,8 +1,76 @@
 const express = require("express");
 const router = express.Router();
-const User = require("../controllers/user");
+const bcrypt = require("bcryptjs");
+const jwt = require("jsonwebtoken");
+const config = require("../config/dev");
+const { check, validationResult } = require("express-validator");
 
-router.post("/register", User.register);
-router.post("/auth", User.authenticate);
+const User = require("../models/user");
 
-module.exports = router;
\ No newline at end of file
+// router.post("/register", User.register);
+// router.post("/auth", User.authenticate);
+
+// @route   POST api/user
```

Comment:
is this node magic that lets you define route url

Reply:
I guess we can delete controller folder if we can do it like this. Noice!

Reply:
> is this node magic that lets you define route url

You mean the comment? It's just for documentation. I think JSDocs can read it, but havent tried it yet.

Reply:
> I guess we can delete controller folder if we can do it like this. Noice!

At least in C#, the BlahController.cs is always where the entry point is for the API. So I think having a separate controller here is not needed. Unless you learn a better way to structure it.

Reply:
oh I thought this would set the url path for the post request below. i.e. '/api/user'.
I structured this router file to only define urls and actual controller method on controller folder and hence, why it was originally router.post("/register", User.register).
My original approach might over complicate stuff though